src/pages/fr/core-concepts/astro-components.md

```diff
@@ -148,9 +150,17 @@ const items = ["Chien", "Chat", "Ornithorynque"];
 
 #### Fragments & valeurs multiples
 
-Souvenez vous : un composant Astro peut faire le rendu de plusieurs éléments sans avoir à les entourer d'une balise `<div>` ou `<>`.
+Un composant Astro peut faire le rendu de plusieurs éléments sans avoir à les entourer d'une balise `<div>` ou `<>`, contrairement au JavaScript ou JSX.
 
-Par contre, quand vous utilisez une expression JSX pour créer dynamiquement plusieurs éléments, vous devez entourer ces éléments d'un **Fragment** comme vous le feriez dans du JavaScript ou du JSX. Astro permet l'utilisation de `<Fragment> </Fragment>` ou des `<> </>`.
+```astro
+---
+// Template avec plusieurs éléments
+---
+<p>Pas besoin d'entourer les éléments avec un élément conteneur.</p>
+<p>Astro supporte plusieurs éléments racines dans son Template.</p>
+```
+
+Par contre, quand vous utilisez une expression pour créer plusieurs éléments, vous devez entourer ces éléments d'un **Fragment** comme vous le feriez dans du JavaScript ou du JSX. Astro permet l'utilisation de `<Fragment> </Fragment>` ou des `<> </>`.
```

Comment:
Noticing “de `<Fragment>`” and “des `<>`”. Should they match?

Reply:
i think it should be the same, i just forget one "s", oops

```suggestion
Par contre, quand vous utilisez une expression pour créer plusieurs éléments, vous devez entourer ces éléments d'un **Fragment** comme vous le feriez dans du JavaScript ou du JSX. Astro permet l'utilisation des `<Fragment> </Fragment>` ou des `<> </>`.
```